daemon/src/engine/qcommon/q_shared.cpp

```diff
@@ -34,6 +34,9 @@ Maryland 20850 USA.
 
 // q_shared.c -- stateless support routines that are included in each code dll
 #include "q_shared.h"
+
+#include <cstddef>     // for std::size_t
```

Comment:
Yes we are basically guaranteed that q_shared includes most of the C++ and C stdlib